source/operations/install-deploy-manage/modify-minio-tenant.rst

```diff
@@ -10,87 +10,59 @@ Modify a MinIO Tenant
+         kubectl apply -k ~/kustomization/TENANT-NAME/
 
-The message adjusts depending on the length of time to expiration:
-   
-- More than 30 days, the message text displays in gray.
-- Within 30 days, the message text changes to orange.
-- Within 10 days, the message text changes to red.
-- Within 24 hours, the message displays as an hour and minute countdown in red text.
-- After expiration, the message displays as ``EXPIRED``.
+      Modify the path to the Kustomization directory to match your local configuration.
 
-.. _minio-k8s-modify-minio-tenant-security:
+   .. tab-item:: Helm
+      :sync: helm
 
-Modify Tenant TLS Configuration
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+      For Helm-deployed Tenants, you can modify the base ``values.yaml`` and upgrade the Tenant using the chart:
 
-The MinIO Operator Console supports adding and removing TLS certificates from a MinIO Tenant.
+      .. code-block:: shell
 
-From the Operator Console view, select the Tenant to open the summary view, then select :guilabel:`Security`.
-You can make the following modifications:
+         helm upgrade  TENANT-NAME minio-operator/tenant -f values.yaml -n TENANT-NAMESPACE
 
-Enable or Disable TLS
-   Toggle the :guilabel:`TLS` switch to direct the Operator to either enable or disable TLS for the deployment.
-   The MinIO Operator automatically generates the necessary TLS certificates using the Kubernetes TLS API.
-   See :ref:`minio-tls-user-generated` for more information.
+      The command above assumes use of the MinIO Operator Chart repository.
+      If you installed the Chart manually or by using a different repository name, specify that chart or name in the command.
 
-Add Custom TLS Certificates
-   MinIO Tenants support `Server Name Indication (SNI) <https://en.wikipedia.org/wiki/Server_Name_Indication>`__, where the MinIO server identifies which certificate to use based on the hostname specified by the connecting client.
-   The MinIO Operator can attach additional TLS certificates to the Tenant to enable SNI-based TLS connectivity.
+      Replace ``TENANT-NAME`` and ``TENANT-NAMESPACE`` with the name and namespace of the Tenant respectively.
+      You can use ``helm list -n TENANT-NAMESPACE`` to validate the Tenant name.
 
-   To customize the TLS certificates mounted on the MinIO Tenant, enable the :guilabel:`Custom Certificates` switch.
-   Select the :guilabel:`Add Certificate +` button to add custom TLS certificates.
+      See :ref:`minio-tenant-chart-values` for more complete documentation on the available Chart fields.
 
-Add Trusted Certificate Authorities
-   The MinIO Tenant validates the TLS certificate presented by each connecting client against the host system's trusted root certificate store.
-   The MinIO Operator can attach additional third-party Certificate Authorities (CA) to the Tenant to allow validation of client TLS certificates signed by those CAs.
+   .. tab-item:: Operator Console
```

Comment:
Why are we keeping this tab?

Reply:
Maybe call it `kubectl` instead of `Operator Console`?

Reply:
Good question actually.

Reply:
Oh the intent was to guide users away from it for existing installs, but that's handled in the upgrade docs.